components/Header/Header.css

```diff
@@ -0,0 +1,10 @@
+.header-subtitle{
+    font-size: 20px;
+    margin-top: 8px;
+}
+
+.header-title {
+    font-size: 40px;
+    display: block;
```

Comment:
What does `display: block` do here?

Reply:
Span acts naturally like inline text. So in order for the title and subtitle to be on top of each other one of the spans need to be told to act like block so how headers and paragraphs act.